scipy/sparse/csgraph/tests/test_matching.py

```diff
@@ -143,3 +144,54 @@ def test_large_random_graph_with_one_edge_incident_to_each_vertex():
     # Should get identity matrix back
     assert_equal(any(C1.diagonal() == 0), False)
     assert_equal(any(C2.diagonal() == 0), False)
+
+
+def test_min_weight_full_matching_empty_graph():
+    biadjacency_matrix = csr_matrix((0, 0))
+    row_ind, col_ind = min_weight_full_bipartite_matching(biadjacency_matrix)
+    assert len(row_ind) == 0
+    assert len(col_ind) == 0
+
+
+def test_min_weight_full_matching_empty_left_partition():
+    biadjacency_matrix = csr_matrix((2, 0))
+    row_ind, col_ind = min_weight_full_bipartite_matching(biadjacency_matrix)
+    assert len(row_ind) == 0
+    assert len(col_ind) == 0
+
+
+def test_min_weight_full_matching_empty_right_partition():
+    biadjacency_matrix = csr_matrix((0, 3))
+    row_ind, col_ind = min_weight_full_bipartite_matching(biadjacency_matrix)
+    assert len(row_ind) == 0
+    assert len(col_ind) == 0
```

Comment:
We could certainly reduce duplication in the tests given same function and same expected row/column values, parametrizing over the different biadjacency matrices, but that could be done later too.

Reply:
Changed to

```python
@pytest.mark.parametrize('num_rows,num_cols', [(0, 0), (2, 0), (0, 3)])
def test_min_weight_full_matching_trivial_graph(num_rows, num_cols):
    ...
```